gui/slick/interfaces/default/home.mako

```diff
@@ -814,6 +818,15 @@ $(document).ready(function(){
             </script>
         </td>
 
+        <% show_size = 0 %>
+        % try:
+            <% show_size = sickbeard.helpers.get_size(curShow.location) %>
+        % except sickbeard.exceptions.ShowDirNotFoundException:
+            <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        % endtry
+
```

Comment:
My kids have a saying... "Daddy always wins!!!" lol

Not trying to scrutinize so hard, just being super vigilant to try and keep a lease on any potential new bugs. This is especially important at this stage because I am looking to make a major release (v4.1.0) once we have a severely stable base.

That release will mark the start of allowing some major changes again.